### PR TITLE
Drawer and styling updates

### PR DESCRIPTION
**Pull request by RyanZim**
Demo at http://better-crib.surge.sh/

### CODE REVIEW

File: app/index.css [hunk 1]
```diff
@@ -15,22 +18,51 @@ body {
...
 
-pre {
-  padding: 1rem;
-  background: #eee;
-  border-radius: 0.3rem;
+.plugins {
+  left: -$drawer-width;
+  width: $drawer-width;
+  background: $dark;
+  overflow-y: scroll;
```
Comment (sylvainpolletvillard) [inline app/index.css]: I have a scrolling issue where sidebar is cut at the bottom. I think we should set `overflow: auto;` for both .container and .plugins

File: app/components/App/App.svelte.html [hunk 2]
```diff
@@ -1,12 +1,32 @@
...
+      <label>
+        <input type="checkbox" checked bind:checked="drawerOpen" />
+        Show Config
+      </label>
+    </div>
+    <div class="input-output">
+      <Ace bind:value="input" id="input" />
+      <Ace bind:value="output" id="output" />
+    </div>
+    <pre class="error {{error ? 'visible' : ''}}">{{ error }}</pre>
```
Comment (sylvainpolletvillard) [inline app/components/App/App.svelte.html]: the error block is always visible

Comment (RyanZim) [reply app/components/App/App.svelte.html]: That's strange; is it just an empty block?

Comment (sylvainpolletvillard) [reply app/components/App/App.svelte.html]: yes, I don't see where you hide it by default

Comment (RyanZim) [reply app/components/App/App.svelte.html]: @sylvainpolletvillard Are you just looking at the code, or are you actually looking at this in the browser? I have the bottom property set to a negative value by default.

Comment (sylvainpolletvillard) [reply app/components/App/App.svelte.html]: https://i.imgur.com/CZNfRqz.jpg

Comment (RyanZim) [reply app/components/App/App.svelte.html]: Oops, sorry; I knew I had it working earlier, but this is indeed broken on my end.

Comment (sylvainpolletvillard) [reply app/components/App/App.svelte.html]: `position:absolute; bottom:-25vh` expands the page height on Chrome, Windows. It works fine with `position: fixed` though

Comment (RyanZim) [reply app/components/App/App.svelte.html]: Yeah, I had used `fixed` earlier, then changed to `absolute` without actually testing this.

**Comment by RyanZim** [comment]
Landed in https://github.com/RyanZim/postcss-repl/commit/39797e890f017a76fcacfe4e2cfe78f2e3928f96.

**Pull request closed without merging**